src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,26 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  closeModal: (todo: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, closeModal }) => {
+  const [user, setUser] = useState<User>();
+
+  useEffect(() => {
+    getUser(todo.userId).then(setUser);
+  }, []);
```

Comment:
The useEffect dependency array is empty, which means the effect will only run once when the component mounts. Since the effect fetches the user based on todo.userId, it is a good idea to add todo.userId to the dependency array to refetch the user when the userId changes.